skins/forward.scss

```
@@ -0,0 +1,10 @@
+.forwardprotocol {
```

Your class name should be similar to file name.
```suggestion
.forward {
```